swarm/network/kademlia.go

```diff
@@ -336,24 +336,32 @@ func (k *Kademlia) EachBin(base []byte, pof pot.Pof, o int, eachBinFunc func(con
 	var i int
 	var startPo int
 	var endPo int
-	kadDepth := int(k.depth)
+	depth := int(k.depth)
 
 	k.conns.EachBin(base, pof, o, func(po, size int, f func(func(val pot.Val, i int) bool) bool) bool {
-		if po < kadDepth {
-			endPo = po
-			if i > 0 {
-				startPo = endPo + 1
-			}
-		} else if endPo < kadDepth || endPo == 0 {
-			if po == 0 && kadDepth == 0 {
-				startPo = endPo
+		if depth == 0 && po == 0 {
```

Comment:
this is incorrect

Reply:
After the review today still incorrect? If yes, what should it be?